kcidb/monitor/test_output.py

```diff
@@ -74,4 +74,15 @@ def test_min(self):
         self.assertEqual(message['X-KCIDB-Notification-ID'], notification.id)
         self.assertEqual(message['X-KCIDB-Notification-Message-ID'], "id")
         self.assertIn("Revision detected: ", message['Subject'])
-        self.assertIn("We detected a new revision!", message.get_payload())
+        text, html = message.get_payload()
+        self.assertEqual('text/plain', text.get_content_type())
+        self.assertEqual('utf-8', text.get_content_charset())
+        content = text.get_payload()
+        if text['content-transfer-encoding'] != '7bit':
```

Comment:
Because of this: https://stackoverflow.com/questions/25710599/content-transfer-encoding-7bit-or-8-bit

Reply:
Should I take it out?

Reply:
Regardless of whichever transfer encoding the Python's email library chooses for us (since we don't specify it explicitly), we should have the "We detected a new revision!" text in there. So we shouldn't skip that check if it's 7-bit transfer encoding, but should check it in both cases. Latin letters and basic punctuation will be represented as is in either 7-bit and 8-bit transfer encoding (because their character codes fit 7-bit, because 7-bit encoding was created to accommodate them), so the check should work in both cases.

So, yes, please remove that condition here and in the other spot, because we want to check the message contents in either case. Thank you!

Reply:
I did in the last commit